voxblox/include/voxblox/utils/camera_model.h

```diff
@@ -61,6 +61,13 @@ class CameraModel {
   void getAabb(Point* aabb_min, Point* aabb_max) const;
   bool isPointInView(const Point& point) const;
 
+  // Accessor functions for visualization (or other reasons).
+  const AlignedVector<Plane>& getBoundingPlanes() const {
+    return bounding_planes_;
+  }
+  // Gives all the bounding lines of the planes in connecting order.
+  void getBoundingLines(AlignedVector<Point>* lines) const;
```

Comment:
It might be helpful to comment on what frame the lines/planes are returned here and above?